boards/arm/mps2_an519/doc/index.rst

```diff
@@ -0,0 +1,231 @@
+.. _mps3_an547_board:
+
+ARM MPS3 AN547
+###############
+
+Overview
+********
+
+The mps3_an547 board configuration is used by Zephyr applications that run
+on the MPS3 AN547 board. It provides support for the MPS3 AN547 ARM Cortex-M55
+CPU and the following devices:
+
+- Nested Vectored Interrupt Controller (NVIC)
+- System Tick System Clock (SYSTICK)
+- Cortex-M System Design Kit GPIO
+- Cortex-M System Design Kit UART
+- Ethos-U55 NPU
+
+.. image:: img/mps3_an547.png
+     :width: 666px
+     :align: center
+     :height: 546px
+     :alt: ARM MPS3 AN547
+
+This board configuration also supports using the `Corstone-300 FVP`_ to emulate
+a MPS3 AN547 hardware platform.
+
+The Corstone-300 FVP (Fixed Virtual Platform) is a complete simulation of the
+Arm system, including processor, memory and peripherals. It is a available free
```

Comment:
```suggestion
Arm system, including processor, memory and peripherals. It is available free
```

Reply:
This part was removed, so this is resolved.